cpp/src/slice2java/Gen.cpp

```diff
@@ -1380,26 +1379,11 @@ Slice::JavaVisitor::writeDispatch(Output& out, const InterfaceDefPtr& p)
         string opName = op->name();
         out << sp;
 
-        out << nl << "/**";
-        out << nl << " * @hidden";
-        out << nl << " * @param obj -";
-        out << nl << " * @param inS -";
-        out << nl << " * @param current -";
-        out << nl << " * @return -";
-        if (!op->throws().empty() || op->hasMetaData("java:UserException") || op->hasMetaData("UserException"))
-        {
-            out << nl << " * @throws " << getUnqualified("com.zeroc.Ice.UserException", package) << " -";
-        }
-        out << nl << "**/";
-        if (dc && dc->isDeprecated())
-        {
-            out << nl << "@Deprecated";
-        }
+        writeHiddenDocComment(out);
         out << nl << "static java.util.concurrent.CompletionStage<"
-            << getUnqualified("com.zeroc.Ice.OutputStream", package) << "> _iceD_" << opName << '(';
+            << getUnqualified("com.zeroc.Ice.OutgoingResponse", package) << "> _iceD_" << opName << '(';
```

Comment:
I'd like to get rid of many of the getUnqualified for fixed com.zeroc.Ice symbols in a follow-up PR.

Reply:
Fixed by #2298.